concord-server/server/src/main/java/com/rabidgremlin/concord/api/UserVoteCount.java

```diff
@@ -0,0 +1,82 @@
+package com.rabidgremlin.concord.api;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+public class UserVoteCount
+{
+  private String userId;
+
+  private int voteCount;
+
+  // Needed for Bean Mapper
+  public UserVoteCount()
+  {
+  }
+
+  public UserVoteCount(String userId, int voteCount)
+  {
+    this.userId = userId;
+    this.voteCount = voteCount;
+  }
+
+  public String getUserId()
+  {
+    return userId;
+  }
+
+  public void setUserId(String userId)
+  {
+    this.userId = userId;
+  }
+
+  public int getVoteCount()
+  {
+    return voteCount;
+  }
+
+  public void setVoteCount(int voteCount)
+  {
+    this.voteCount = voteCount;
+  }
+
+  @Override
+  public String toString()
+  {
+    return "UserVoteCount{" +
```

Comment:
Consider using ToStringBuilder https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ToStringBuilder.html

Reply:
could.. I generated this class with IntelliJ